miden-lib/asm/miden/account.masm

```diff
@@ -101,6 +101,37 @@ export.set_item
     # => [R', V]
 end
 
+#! Gets a map item from the account storage. Panics if the index is out of bounds.
+#!
+#! Stack: [index, KEY]
+#! Output: [VALUE]
+#!
+#! - index is the index of the item to get.
+#! - VALUE is the value of the item.
+export.get_map_item
+    push.0 movdn.5 push.0 movdn.5 push.0 movdn.5
```

Comment:
These seem unnecessary, the signature of `get_account_map_item` is currently:

https://github.com/0xPolygonMiden/miden-base/blob/29ab9799ab333cb246ff6dd9d582ae600206808a/miden-lib/asm/kernels/transaction/api.masm#L196-L197

So the existing stack should be good enough.